pkg/cmd/kind/load/docker-image/docker-image.go

```diff
@@ -145,7 +145,7 @@ func runE(logger log.Logger, flags *flagpole, args []string) error {
 				// We will try to re-tag the image. If the re-tag fails, we will fall back to the default behavior of loading
 				// the images into the nodes again
 				logger.V(0).Infof("Image with ID: %s already present on the node %s but is missing the tag %s. re-tagging...", imageID, node.String(), imageName)
```

Comment:
we should use `sanitizedImageName`

Reply:
I originally opted to respect what the user provided as an arg but this would be more accurate. updated.